frontend/src/components/ConfigForm.vue

```diff
@@ -476,6 +488,11 @@ export default {
     }
   },
   methods: {
+    async clearSelectedAcronymDetails() {
+      await this.$store.dispatch('configForm/getAcronymDetails', null);
```

Comment:
This doesn't need the async/await from what it looks like.

Reply:
Fixed